usaspending_api/awards/models_matviews.py

```diff
@@ -288,14 +264,11 @@ class Meta:
         db_table = 'summary_view'
 
 
-class SumaryNaicsCodesView(models.Model):
-
-    # Fields
+class SummaryNaicsCodesView(models.Model):
```

Comment:
Since there are some matview updates to this file (for the models and associated use thereof), @tony-sappe can you please verify that this will not interfere with anything anywhere else?

Reply:
Looks like those matview updates were an artifact of this branch being behind dev - when I merged dev back in, they disappeared from the diff (they're certainly nothing I wrote)